From bee3b5fcb73c6abe37a3447a67c728cb08e251e3 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Thu, 17 Aug 2017 17:59:05 -0500
Subject: [PATCH] add missing rule to makefile

The compilation from asm to object was not included and brings issues
when build a rpm package on Clear Linux*

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
---
 Make.rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Make.rules b/Make.rules
index 5b1c286..08c489e 100644
--- a/Make.rules
+++ b/Make.rules
@@ -51,6 +51,9 @@
 %.o: %.c
 	$(CC) $(INCDIR) $(CFLAGS) $(CPPFLAGS) -c $< -o $@
 
+%.o: %.S
+	$(CC) $(ASFLAGS) $(INCDIR) $(CFLAGS) $(CPPFLAGS) -c $< -o $@
+
 %.S: %.c
 	$(CC) $(INCDIR) $(CFLAGS) $(CPPFLAGS) -S $< -o $@
 
-- 
2.14.1

